Cargo.toml

```diff
@@ -1,2 +1,5 @@
 [workspace]
 members = ["capsicum", "casper-sys"]
+
+[patch.crates-io]
+libnv-sys = { git = "https://github.com/asomers/libnv-rs", rev = "597c78d" }
```

Comment:
Do we need the patched crate, or did you mean to remove this before submitting?

Reply:
Yes, we required the patched libnv.  It just got published last night, so I updated this PR to use the released version.